csrc/runtime/executor_dispatch.h

```diff
@@ -16,14 +16,20 @@ namespace nvfuser {
 // ExprEvalExecutor
 class ExecutorDispatch {
  public:
-  // Iterates through executors in priority order creating the first executor
-  // that returns true when checking their "supported" method
+  // If `scheduler_type` is `SchedulerType::None`, this function Iterates
+  // through executors in priority order creating the first executor that
+  // returns true when checking their "supported" method. Otherwise, create the
+  // executor according to `scheduler_type`, which is faster.
+  //
+  // The slow path (i.e. `SchedulerType::None`) is only used by
+  // MultiDeviceExecutor at this moment.
   static std::unique_ptr<ExecutorAbstract> makeExecutor(
       Fusion* fusion,
       int64_t fusion_id = -1,
       int64_t concrete_id = -1,
       int64_t runtime_id = -1,
-      int64_t group_id = -1);
+      int64_t group_id = -1,
+      SchedulerType scheduler_type);
```

Comment:
```suggestion
      SchedulerType scheduler_type = SchedulerType::None);
```
**[Suggested by AI]** The change adds a default value (`SchedulerType::None`) to the `scheduler_type` parameter in a function definition.

**Summary:** This likely fixes a build error caused by the function being called without providing a value for the `scheduler_type` parameter. By providing a default value, the function can now be called without explicitly specifying the scheduler type, resolving the error. It makes the parameter optional.